templates/default/authentication.html.twig

```diff
@@ -45,6 +45,34 @@
 
     <script>
         window.onload = function(){
+            const form = document.getElementById('form');
+            const setText = (id, value) => {
+                const el = document.getElementById(id);
+                if (el) el.textContent = value ?? '';
```

Comment:
Had to spend a minute parsing this line. Would be more readable with brackets, but functionally fine.

Reply:
I will leave it now as-as.